listener_test.go

```diff
@@ -144,11 +146,11 @@ var _ = Describe("listener", func() {
 			Expect(ds.Scope).To(Equal("s2"))
 
 			scopes := findScopesForId("a")
-			Expect(len(scopes)).To(Equal(2))
-			Expect(scopes[0]).To(Equal("s1"))
-			Expect(scopes[1]).To(Equal("s2"))
-
-			//restore the last snapshot
+			Expect(len(scopes)).To(Equal(6))
+			expectedScopes := []string{"s1", "s2", "org_scope_1", "env_scope_1", "env_scope_2", "env_scope_3"}
```

Comment:
That's a good point. Not done in mgmt. What I assume is not about prefix. What I assume is org_scope, scope and env_scope cannot be same, which means the "UNION" will never deduplicate an org_scope, an env_scope and a scope.